@here/harp-datasource-protocol/tsconfig.json

```diff
@@ -2,13 +2,27 @@
     "compilerOptions": {
         "target": "es2017",
         "module": "commonjs",
+        "outDir": "dist/cjs",
```

Comment:
This changes layout of files so `@harp-mapview/lib/MapView.` will become `@harp-mapview/dist/*/lib/MapView` of course, people usually import from module `index`, but they need to be warned. 

#release-notes-needed ?